backend/data/src/main/kotlin/io/tolgee/service/apps/AppManifestValidator.kt

```
@@ -0,0 +1,211 @@
+  private fun currentRequestOrigin(): String? {
+    val attributes = RequestContextHolder.getRequestAttributes() as? ServletRequestAttributes ?: return null
+    val request = attributes.request
+    return parseAbsoluteHttpUrl("${request.scheme}://${request.serverName}:${request.serverPort}")
+      ?.let { originOf(it) }
+  }
+
+  private fun parseAbsoluteHttpUrl(value: String): URI? {
+    val uri =
+      try {
+        URI(value)
+      } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)